code/modules/clothing/clothing.dm

```diff
@@ -161,7 +161,7 @@
 /obj/item/clothing/obj_break(damage_flag)
 	if(!damaged_clothes)
 		update_clothes_damaged_state(TRUE)
-
+	to_chat(usr, "<span class='notice'>Your [src] starts to fall apart!")

```

Comment:
when the jumpsuit's health is below integrity_failure level. so once. right now there's no message, just the sprite change